blocks/init/src/Blocks/components/layout-three-columns/manifest.json

```diff
@@ -24,6 +24,10 @@
 			"type": "string",
 			"default": "div"
 		},
+		"layoutThreeColumnsAriaRole": {
+			"type": "string",
+			"default": ""
```

Comment:
Is `default` property needed if it's empty?

Reply:
@mbmjertan can you fix this so we can merge this pr?

Reply:
Will do, @iruzevic! :)